util/tls/test/README.md

```diff
@@ -0,0 +1,4 @@
+Generate these files by running:
+```
+openssl req -newkey rsa:2048 -new -nodes -x509 -days 1 -out test/cert.crt -keyout test/key.key -subj "/C=US/ST=California/L=South San Francisco/O=Stripe/OU=Veneur/CN=localhost"
```

Comment:
Make a script instead of a readme?

Reply:
I think similar scripts live in the `testdata/_bin` directory. Not sure if it makes sense to put that there.

Reply:
Ah thank you @mimran-stripe, I'm using those instead.